ci-operator/config/openstack-k8s-operators/octavia-operator/openstack-k8s-operators-octavia-operator-main.yaml

```diff
@@ -0,0 +1,48 @@
+base_images:
+  golangci-lint:
+    name: golangci-lint
+    namespace: ci
+    tag: v1.46.0
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
```

Comment:
Could we please update this tag to: rhel-8-release-golang-1.18-openshift-4.12

Reply:
Changed. Thanks